work/v1/0000_00_work.open-cluster-management.io_manifestworks.crd.yaml

```diff
@@ -75,7 +75,6 @@ spec:
                     description: ManifestConfigOption represents the configurations of a manifest defined in workload field.
                     type: object
                     required:
-                      - feedbackRules
```

Comment:
is this intended change?

Reply:
This allows that user only set updateStrategy but not feedbackRules.